src/components/Attachments/EditAttachments.tsx

```diff
@@ -101,7 +100,7 @@ const EditAttachments = ({
                 }
             }
         })
-    }, [documentId, documentType, fetchData])
+    }, [documentId, documentType, fetchData, componentPayload, setComponentPayload])
```

Comment:
**componentPayload* and **setComponentPayload** should be remove from useEffect's dependencies. They can cause some logic errors

Reply:
@heliocastro I know you're trying to fix the warnings, but useEffect's dependencies should not be changed.